tests/specs/npm/info_peer_deps_json/peer_deps_with_copied_folders/main_info_json.out

```diff
@@ -59,39 +69,45 @@
       "dependencies": [
         "@denotest/peer-dep-test-grandchild@1.0.0_@denotest+peer-dep-test-peer@1.0.0",
         "@denotest/peer-dep-test-peer@1.0.0"
-      ]
+      ],
+      "registryUrl": "http://localhost:4260/"
     },
     "@denotest/peer-dep-test-child@2.0.0_@denotest+peer-dep-test-peer@2.0.0": {
       "name": "@denotest/peer-dep-test-child",
       "version": "2.0.0",
       "dependencies": [
         "@denotest/peer-dep-test-grandchild@1.0.0_@denotest+peer-dep-test-peer@2.0.0",
         "@denotest/peer-dep-test-peer@2.0.0"
-      ]
+      ],
+      "registryUrl": "http://localhost:4260/"
     },
     "@denotest/peer-dep-test-grandchild@1.0.0_@denotest+peer-dep-test-peer@1.0.0": {
       "name": "@denotest/peer-dep-test-grandchild",
       "version": "1.0.0",
       "dependencies": [
         "@denotest/peer-dep-test-peer@1.0.0"
-      ]
+      ],
+      "registryUrl": "http://localhost:4260/"
     },
     "@denotest/peer-dep-test-grandchild@1.0.0_@denotest+peer-dep-test-peer@2.0.0": {
       "name": "@denotest/peer-dep-test-grandchild",
       "version": "1.0.0",
       "dependencies": [
         "@denotest/peer-dep-test-peer@2.0.0"
-      ]
+      ],
+      "registryUrl": "http://localhost:4260/"
     },
     "@denotest/peer-dep-test-peer@1.0.0": {
       "name": "@denotest/peer-dep-test-peer",
       "version": "1.0.0",
-      "dependencies": []
+      "dependencies": [],
+      "registryUrl": "http://localhost:4260/"
     },
     "@denotest/peer-dep-test-peer@2.0.0": {
       "name": "@denotest/peer-dep-test-peer",
       "version": "2.0.0",
-      "dependencies": []
+      "dependencies": [],
+      "registryUrl": "http://localhost:4260/"
```

Comment:
Maybe we should add a test that uses an npmrc so we can ensure this is actually uses the npmrc and not just the npm url?